CHANGELOG.md

```diff
@@ -32,6 +32,10 @@
 * Make `redundant_objc_attribute` rule autocorrectable.  
   [Daniel Metzing](https://github.com/dirtydanee)
 
+* `nimble_operator` now warns about `beTrue()` and `beFalse()`
```

Comment:
requires two trailing spaces after a period as described in CONTRIBUTING.md: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes